handlers/api.go

```diff
@@ -2340,7 +2340,7 @@ func ApiValidatorAttestations(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	history, err := db.BigtableClient.GetValidatorAttestationHistory(queryIndices, services.LatestEpoch()-101, services.LatestEpoch())
+	history, err := db.BigtableClient.GetValidatorAttestationHistory(queryIndices, services.LatestEpoch()-100, services.LatestEpoch())
```

Comment:
Is is meant to select the last 101 epochs instead of 100?

Reply:
For this BIDS I left the behaviour how it was but I also asked this in BIDS-2189 and it seems both the summary and response amount is wrong.
Fixed.